### PR TITLE
[hacktoberfest] Chisend: Twitter Bot for chimoney

### PR DESCRIPTION
**Pull request by thelimeskies**
Added docker 
Added new features 
Tested
Deployment in progress

### CODE REVIEW

**Comment by uchibeke** [comment]
hi @thelimeskies could you open 2 PRs, one for each submission? This will enable us count both submission individually and give you the prizes

Also its easier to review

**Comment by thelimeskies** [comment]
Hi @uchibeke I can only make one PR per branch and every push I make updates the PR.

**Comment by uchibeke** [comment]
@Regedit-msc could you help here?

File: submissions/Chisend/bots/temp.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
+# this is a code playground for testing things out
```
Comment (Regedit-msc) [inline submissions/Chisend/bots/temp.py:1]: Is this file still in use?

**Review by Regedit-msc — COMMENTED** [review]
Great job once again! Please look through the comments and you good!

File: submissions/Chisend/bots/oauth-key-gen.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# 3 legged OAuth
+
+import tweepy
+
+oauth1_user_handler = tweepy.OAuth1UserHandler(
+    "access-token", "access-secret", callback="oob"
+)
+
+print(oauth1_user_handler.get_authorization_url(signin_with_twitter=True))
```
[the remark this answers is not in the document]
Comment (thelimeskies) [reply submissions/Chisend/bots/oauth-key-gen.py:9]: This is a utility for generating access keys not part of the main package, It was a custom requirement so you could use another account other than your Twitter developer account for the bot.
It is more like a script needed for setup(one-time use).

File: submissions/Chisend/bots/temp.py (hunk 1 above)
Comment (thelimeskies) [reply submissions/Chisend/bots/temp.py:1]: No, i thought I ignored it. will update the gitignore to reflect it.

File: submissions/Chisend/bots/oauth-key-gen.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Regedit-msc) [reply submissions/Chisend/bots/oauth-key-gen.py:9]: Essentially the `print` statement is relevant then. Alright cool. The loggers?

File: submissions/Chisend/bots/temp.py (hunk 1 above)
Comment (Regedit-msc) [reply submissions/Chisend/bots/temp.py:1]: Alright please check it out.

File: submissions/Chisend/bots/oauth-key-gen.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (thelimeskies) [reply submissions/Chisend/bots/oauth-key-gen.py:9]: Most of the logs I added were for monitoring and tracking incase of error during use so I can trace what the user requested and the action performed with a time stamp. Basically going to be needed for customer service. Then for service improvement based on the logs I could know what to update. They would be moved to web in future version

File: submissions/Chisend/bots/temp.py (hunk 1 above)
Comment (thelimeskies) [reply submissions/Chisend/bots/temp.py:1]: Can I handle this later? So do it with the next feature and PR I open. I've added part of the giveaway feature but haven't pushed that yet.

Comment (Regedit-msc) [reply submissions/Chisend/bots/temp.py:1]: Alright. Can you finish up what you have to do then open the pr again. In the meantime you can move it to draft then nudge me once you set.

Comment (thelimeskies) [reply submissions/Chisend/bots/temp.py]: Done.

Removed the temp.py

**Comment by thelimeskies** [comment]
The removal was a mistake.

**Review by Regedit-msc — APPROVED** [review]
LGTM👍🏾!

**Pull request merged**